code/backend/batch/utilities/helpers/config/config_helper.py

```diff
@@ -222,7 +222,7 @@ def validate_config(config: dict):
                 and unsupported_advanced_image_processing_file_type
             ):
                 raise Exception(
-                    f"Advanced image processing has been enabled for document type {document_type}, but only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
+                    f"Advanced image processing has not been enabled for document type {document_type}, but only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
```

Comment:
I think a small change will make this read even better:
```suggestion
                    f"Advanced image processing has not been enabled for document type {document_type}, as only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
```

Reply:
I added this change